asv_bench/benchmarks/benchmarks.py

```diff
@@ -218,12 +215,11 @@ class TimeBinaryOp:
     ]
 
     def setup(self, data_size, binary_op, axis):
-        # shape for generate_dataframe: first - ncols, second - nrows
         self.df1 = generate_dataframe(
-            ASV_USE_IMPL, "int", data_size[1], data_size[0], RAND_LOW, RAND_HIGH
+            ASV_USE_IMPL, "int", data_size[0], data_size[1], RAND_LOW, RAND_HIGH
         )
         self.df2 = generate_dataframe(
-            ASV_USE_IMPL, "int", data_size[3], data_size[2], RAND_LOW, RAND_HIGH
+            ASV_USE_IMPL, "int", data_size[2], data_size[3], RAND_LOW, RAND_HIGH
```

Comment:
It's actually not very easy to follow the meaning of indices of the `data_size` array. We can create some data-class called `Shape` which will contain only two fields: `nrows` and `ncols` and since then we can fill `data_size` with the instances of this class. It would be much easier to understand this syntax of generating frames using this class, what do you think?
```python
self.df1 = generate_dataframe(
    ASV_USE_IMPL, "int", data_size[0].nrows, data_size[0].ncols, RAND_LOW, RAND_HIGH
)
self.df2 = generate_dataframe(
    ASV_USE_IMPL, "int", data_size[1].nrows, data_size[1].ncols RAND_LOW, RAND_HIGH
)
```

Reply:
When we get the size of the dataframe, it is of type tuple. So I think there is enough to use tuple, but with some small changes(No need to create extra entities). Please review.

Reply:
Okay, this looks good.